### ISSUE TITLE
Not working at all.

### ISSUE THREAD

**Issue by maxrr** [noxwyll/Steam-Login-MyBB-1.8#66]
tl;dr: MyBB 1.8.12. Forums use default login when they shouldn't.

I'm using MyBB 1.8.12, and our forums have been working flawlessly for months.. Early today they were completely operational, but after I got home, I was told by a community member that the login feature on our forums is "broken". When I asked them to elaborate, they said they could not sign in. I was confused, went to our website, and found out that the 'Login via Steam' button links to the default (for our theme) login page. I have made no intentional modifications to any part of the website between this morning and now. I have tried countless times to replace the files themselves, as well as activating/deactivating the plugins from the admin control panel. Any help is appreciated.

**Comment by TsunaMoo** [comment]
Strange i run it on this version fine. I think i will need more info from you or access to check what is going on.
